manifests/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -237,23 +228,6 @@ spec:
 {{ toYaml $gateway.additionalContainers | indent 8 }}
 {{- end }}
       volumes:
-      {{- if .Values.global.sds.enabled }}
-      - name: sdsudspath
-        hostPath:
-          path: /var/run/sds
-      - name: istio-token
```

Comment:
I think we still need to mount istio-token, regardless of nodeagent or istioagent right?

Reply:
Good point... We have the same logic in some other place in the template in the other deployments. I think Egress has some issues, we need to do some more testing for that

Reply:
I think we need to do the same as ingress, right? the egress gateway doesn't use the K8s 3P JWT?